osmo_jupyter/plot.py

```diff
@@ -153,6 +153,26 @@ def _add_dicts(dicts):
     return output_dict
 
 
+def _get_colors_with_pattern(pattern):
```

Comment:
nit: `get` seems like a slight misnomer here as it's really constructing a thing vs. getting a thing. 

What's your rationale for requiring the '*' vs just providing a postfix? e.g. _get_colors_with_pattern('_mean')? Are there valid cases where the color is in the middle of the string? If so, maybe update the example to use that.

Reply:
not really, it was mainly designed to support a prefix or a suffix, and the middle thing was just a side effect. we seem to have since landed on more suffix-y patterns since then. The `'*'` pattern a tad more user-friendly to me, and there's a slight concern for backwards compatibility too (with existing uses of `get_rgb_columns()`, so I'm leaning towards keeping this as-is.
if you still disagree I can change it.

Reply:
re: misnomer: huh, I often use `get_` as a prefix to verbify my functions. I do it to avoid `colors_with_pattern = colors_with_pattern()' aliasing. `construct` seems a little excessive, though. do you have a different suggestion?